src/SprykerEco/Client/Computop/ComputopClientInterface.php

```diff
@@ -101,6 +100,30 @@ public function savePayNowInitResponse(QuoteTransfer $quoteTransfer);
      */
     public function savePayPalInitResponse(QuoteTransfer $quoteTransfer);
 
+    /**
+     * Specification:
+     * - Persists `PayPal Express init` response to the database.
```

Comment:
Usually, we add a line `Makes Zed request` to methods where we call ZedStub.

Reply:
Added below